doc/guides/maintaining-the-build-files.md

```diff
@@ -34,6 +34,11 @@ update the build and testing process on Windows.
 
 ## configure
 
+The `configure` script recognizes many CLI flags for special build formulas. Many are not represented by `vcbuild` shortcuts, and need to be passed either by:
```

Comment:
Please wrap line so it's < 80 char long.
BTW: you can run the markdown linter by calling
```cmd
node tools/lint-md.js -q -f doc/guides/
```